app/controllers/api/stages_controller.rb

```diff
@@ -18,10 +16,19 @@ def clone
     end
   end
 
+  def duplicable
+    render json: @project.duplicable_stage
+  end
+
+  def put_duplicable
```

Comment:
Sorry. `#put_duplicable` is to change which stage is marked duplicable. Only one stage per project is "duplicable". The `#duplicable` method is simply the currently marked stage on the project that is duplicable. Clone is just clone.